src/components/Slider/SliderBase/SliderBase.tsx

```diff
@@ -54,19 +56,19 @@ const SliderBase = forwardRef(({ className }, ref) => {
     }
     if (isPageUpEvent(e)) {
       e.preventDefault();
-      return increaseValue(calculatePageStep({ min, max, step }));
+      return increaseValue(calculatePageStep(max, min, step));
     }
     if (isPageDownEvent(e)) {
       e.preventDefault();
-      return decreaseValue(calculatePageStep({ min, max, step }));
+      return decreaseValue(calculatePageStep(max, min, step));
     }
     if (isHomeEvent(e)) {
       e.preventDefault();
-      return changeThumbValue(max);
+      return changeThumbValue(min);
```

Comment:
Did you intentionally change max to min or is it a mistake?

Reply:
Intentionally made this and forgot to mention - I think it was bugged before. Home button should make it to 0 and end button should make it to 100 - that's logical, but it was the other way around before.
It's even more obvious on Mac, fn + arrow left made it to 100 and fn + arrow right made it to 0 - not correct

Reply:
Agree! 👍